assets/app/styles/_core.less

```diff
@@ -331,6 +331,96 @@ code {
   overflow-x: auto; // remove when we update to latest patternfly with twbs 3.3.4
 }
 
+.table-events {
+  background-color: #fff;
+  > tbody {
+    + tbody {
+      border-top-width: 1px;
+    }
+  }
+  tr {
+    th,td {
+      .column-combine {
+        display: none;
+      }
+    }
+    @media (min-width: @screen-lg-min) {
+      td[data-title="Name"] {
+        white-space: nowrap;
+      }
+    }
+  }
+  // Combine event reason and message into one column between 768 and 1100, so that table doesn't cause horizontal scroll and message is readable
+  @media (min-width: @screen-sm-min) and (max-width: @screen-md-max) {
+    > thead {
+      > tr {
+        > th {
+          &:nth-child(4) {
+            display: none;
+          }
+          &:nth-child(5) {
+            .column-combine {
+              display: inline;
+            }
+          }
+        }
+      }
+    }
+    > tbody {
+      > tr {
+        > td {
+          &:nth-child(4) {
+            display: none;
+          }
+          &:nth-child(5) {
+            .column-combine {
+              display: block;
+            }
+          }
+        }
+      }
+    }
+  }
+}
+
+@media (max-width: @screen-xs-max) {
+  .table-events, thead, tbody, th, td, tr {
+    display: block;
+  }
+  .table-events {
+    thead tr {
+      /* Hide table headers (but not display: none;, for accessibility) */
+      position: absolute;
+      top: -9999px;
+      left: -9999px;
+    }
+    > tbody {
+      > tr {
+        > td {
+           /* Behave like a row */
+          border: none;
+          border-bottom: 1px solid #eee;
+          padding: 3px 6px 2px 35%;
+          position: relative;
+          &:last-child {
+            border-bottom: none;
+          }
+          &:before {
+            /* Act as mobile table header */
+            content: attr(data-title);
```

Comment:
I think this will be read by a screen reader... Might be better to just use `display: none` for `thead tr` and let the screen reader use this as the label.